backend/settings.py

```diff
@@ -146,16 +146,19 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': config('MYSQL_DATABASE'),
-        'USER': config('MYSQL_USER'),
-        'PASSWORD': config('MYSQL_PASSWORD'),
-        'HOST': config('MYSQL_HOST'), 
-        'PORT': config('MYSQL_PORT'), 
+        'NAME': config('MYSQL_DATABASE', 'clrt_local'),
+        'USER': config('MYSQL_USER', 'clrt_user'),
+        'PASSWORD': config('MYSQL_PASSWORD', 'clrt_pwd'),
+        'HOST': config('MYSQL_HOST', 'clrt_mysql_host'), 
+        'PORT': config('MYSQL_PORT', '3306'), 
+        'OPTIONS': {
+                'charset': 'utf8mb4',
+        },
```

Comment:
This should resolve the error `?: (django_mysql.W003) The character set is not utf8mb4 for database connection 'default'
8`

Reply:
We should ensure every project we develop supports UTF8, because we never know how our applications will be used.  That character set includes scripts for many languages aside from the Latin one used for English.  Without UTF8 support, some other languages cannot be used in the app.